ui/pages/settings/snaps/view-snap/view-snap.js

```diff
@@ -190,13 +214,48 @@ function ViewSnap() {
             </Text>
           </Button>
           <SnapRemoveWarning
-            isOpen={isShowingRemoveWarning}
+            isOpen={isShowingRemoveWarning && !isKeyringSnap}
```

Comment:
is the second condition needed? isKeyringSnap is not set to `true` otherwise.

Reply:
The second condition is to show the original snap removal confirmation if its not a keyring snap